packages/gatsby-remark-graphviz/src/index.js

```diff
@@ -28,7 +28,8 @@ module.exports = async ({ markdownAST }, pluginOptions = {}) => {
         // Mutate the current node. Converting from a code block to
         // HTML (with svg content)
         node.type = `html`
-        node.value = svgString
+        node.value =
+          `<div class="gatsby-graphviz-container"` + svgString`</div>`
```

Comment:
Shouldn't this be either 
`<div class="gatsby-graphviz-container">${svgString}</div>`
or 
`<div class="gatsby-graphviz-container">` + svgString + `</div>
?

Reply:
first one please ;)
```suggestion
          `<div class="gatsby-graphviz-container">${svgString}</div>`
```